api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/ldap/AbstractLdapProperties.java

```diff
@@ -41,6 +41,22 @@
      * by the underlying Java platform.
      */
     private String keystoreType;
+
+    /**
+     * Name of the truststore to use for the SSL or startTLS connection.
+     */
+    private String trustStore;
+
+    /**
+     * Password needed to open the truststore.
+     */
+    private String trustStorePassword;
+
+    /**
+     * Truststore type.
```

Comment:
Could we outline what the accepted types are, in Javadoc with a brief explanation of what each might do? This stuff comes in very handy inside the CAS Shell and later on as we build UIs on top of properties, etc.